concept/type/impl/TypeImpl.java

```diff
@@ -193,6 +175,7 @@ public final boolean isRemote() {
         public final void setLabel(String label) {
             execute(ConceptProto.Type.Req.newBuilder()
                     .setTypeSetLabelReq(ConceptProto.Type.SetLabel.Req.newBuilder().setLabel(label)));
+            this.label = label;
```

Comment:
`setLabel` was updating the concept on the server, but not updating the local field, `label`. This change fixes that.